ui/src/app/join-lobby-link/join-lobby-link.component.html

```diff
@@ -0,0 +1,13 @@
+<h3>Join lobby!</h3>
+<div class="container">
+    <form class="join-lobby-form" [formGroup]="joinLobbyForm">
+        <mat-form-field class="join-lobby-form_field">
+            <mat-label>Your name: </mat-label>
+            <input matInput formControlName="userName" placeholder="Name" cdkFocusInitial>
+        </mat-form-field>
```

Comment:
As you remove the form group (comment below) in Component's class – don't forget to remove reference to it in the template.
Also you would need to replace `formControlName` with `[formControl]`

Read more here: https://angular.io/guide/reactive-forms#getting-started